src/api/tests/feature_tests/features/createCpmProduct.success.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Create CPM Product - success scenarios
+  These scenarios demonstrate successful Device creation
+
+  Background:
+    Given "default" request headers:
+      | name          | value   |
+      | version       | 1       |
+      | Authorization | letmein |
+
+  Scenario Outline: Successfully create a CPM Product
+    Given I have already made a "POST" request with "default" headers to "Organization" with body:
+      | path                     | value                                                          |
+      | resourceType             | Organization                                                   |
+      | identifier.0.system      | connecting-party-manager/product-team-id                       |
+      | identifier.0.value       | ${ uuid(1) }                                                   |
+      | name                     | My Great Product Team                                          |
+      | partOf.identifier.system | https://directory.spineservices.nhs.uk/ORD/2-0-0/organisations |
+      | partOf.identifier.value  | F5H1R                                                          |
+    When I make a "POST" request with "default" headers to "CpmProduct" with body:
+      | path            | value            |
+      | product_team_id | ${ uuid(1) }     |
+      | product_name    | My Great Product |
+    Then I receive a status code "201" with body
+      | path                             | value                                                               |
+      | resourceType                     | OperationOutcome                                                    |
+      | id                               | << ignore >>                                                        |
+      | meta.profile.0                   | https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome |
+      | issue.0.severity                 | information                                                         |
+      | issue.0.code                     | informational                                                       |
+      | issue.0.details.coding.0.system  | https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome |
+      | issue.0.details.coding.0.code    | RESOURCE_CREATED                                                    |
+      | issue.0.details.coding.0.display | Resource created                                                    |
+      | issue.0.diagnostics              | Resource created                                                    |
+    And the response headers contain:
+      | name           | value            |
+      | Content-Type   | application/json |
+      | Content-Length | 460              |
+
```

Comment:
Add comment: This is template for device that will be used in the read Endpoint.